### PR TITLE
Revert "Update tipsy.py to use sensors service (#16)"

### PR DESCRIPTION
**Pull request by npentrel**
This reverts commit bc0798243bf5cade06587d88b739ca7b65d7e97b.

### CODE REVIEW

**Review by HipsterBrown — APPROVED** [review]
LGTM 👍

File: tipsy-bot/tipsy.py [hunk 1]
```diff
@@ -85,14 +85,13 @@ async def person_detect(detector: VisionClient, sensors: list[Sensor], sensors_s
 async def main():
     robot = await connect()
     base = Base.from_robot(robot, base_name)
-    sensors_svc = SensorsClient.from_robot(robot, name=sensor_service_name)
-    sensors = await sensors_svc.get_sensors()
-    detector = VisionClient.from_robot(robot, name=detector_name)
+    sensors = [Sensor.from_robot(robot, sensor_name) for sensor_name in sensor_names]
+    detector = VisionClient.from_robot(robot, "myPeopleDetector")
```
Comment (mestcihazal) [inline tipsy-bot/tipsy.py]:
```suggestion
    detector = VisionClient.from_robot(robot, name=detector_name)
```

File: tipsy-bot/tipsy.py [hunk 2]
```diff
@@ -15,11 +14,8 @@
 base_name = os.getenv("ROBOT_BASE") or "tipsy-base"
 # change this if you named your camera differently in your robot configuration
 camera_name = os.getenv("ROBOT_CAMERA") or "cam"
-# change this if you named your detector differently in your robot configuration
-detector_name = os.getenv("ROBOT_DETECTOR") or "detect"
-# change this if you named your sensor service differently in your robot configuration
-sensor_service_name = (os.getenv("ROBOT_SENSORS") or "sensors")
-# change this if you named your sensor service differently in your robot configuration
+# change this if you named your sensors differently in your robor configuration
+sensor_names = (os.getenv("ROBOT_SENSORS") or "ultrasonic,ultrasonic2").split(",")
```
Comment (mestcihazal) [inline tipsy-bot/tipsy.py:18]:
```suggestion
sensor_names = (os.getenv("ROBOT_SENSORS") or "ultrasonic,ultrasonic2").split(",")
# change this if you named your detector differently in your robot configuration
detector_name = os.getenv("ROBOT_DETECTOR") or "myPeopleDetector"
```

**Review by mestcihazal — APPROVED** [review]
globally defining detector for consistency

**Pull request merged**